pkg/acsengine/pki.go

```diff
@@ -110,7 +110,6 @@ func createCertificate(commonName string, caCertificate *x509.Certificate, caPri
 		template.KeyUsage |= x509.KeyUsageCertSign
 		template.IsCA = isCA
 	} else if isServer {
-		extraFQDNs = append(extraFQDNs, FQDN)
```

Comment:
Implement either of two recommendations:
1. keep your change, but then remove FQDN through full call stack, or
2. undo your change, and in setDefaultCerts remove the "*", and send in element 0 as the FQDN and [1:] as the FQDNs